src/createReconciler.lua

```diff
@@ -100,6 +122,18 @@ local function createReconciler(renderer)
 					virtualNode.legacyContext
 				)
 
+				-- If updating this node has caused a component higher up the tree to re-render
+				-- and updateChildren to be re-entered for this virtualNode then
+				-- this result is invalid and needs to be disgarded.
+				if config.tempFixUpdateChildrenReEntrancy then
+					if virtualNode.updateChildrenCount ~= currentUpdateChildrenCount then
+						if childNode then
```

Comment:
Confused about this... `childNode` is defined above on line 117, is it even possible for this condition to be false?

Reply:
mountVirtualNode can return nil if the element is a boolean